pkgs/development/libraries/spice/default.nix

```diff
@@ -24,6 +24,16 @@
 , orc
 }:
 
+let
+  # This file was mistakenly not included with the 0.15.0 release.
+  # Should be fixed with the next release.
+  # https://gitlab.freedesktop.org/spice/spice/-/issues/56
+  doxygen_sh = fetchurl {
+    url = "https://gitlab.freedesktop.org/spice/spice/-/raw/0.15/doxygen.sh";
```

Comment:
0.15 is a branch name so potentially unstable, should be `v0.15.0`.

Reply:
Good catch, thanks.